docs/Reference/faq.md

```diff
@@ -28,12 +28,27 @@ Device Management Toolkit aligns to the Intel Network and Edge (NEX) Group suppo
 
 <br>
 
-### How do I migrate versions to a new release?
+### How do I migrate to a new release of Cloud-Deployment?
 
 Resources and information for migrating releases for either a Kubernetes deployment or local Docker deployment can be found in the [Upgrade Toolkit Version documentation](../Deployment/upgradeVersion.md).
 
 <br>
 
+### Will there be any changes to Cloud-Deployment services following the release of Console v1.0?
+
+No changes are currently planned. We will continue supporting our Cloud-Deployment services.
+
+<br>
+
+### What is the long-term vision for MPS and RPS services following the release of Console v1.0?
+
+- Our long-term vision is for Console to support both Enterprise and Cloud deployments, reducing the number of services we need to maintain.
+- As part of this vision, Console will take on the functions of both MPS and RPS. It will handle AMT connections and expose REST APIs to enable AMT features, similar to MPS. It will also manage profile creation, similar to RPS.
+- Furthermore, RPC-Go will be utilized for the configuration and activation of AMT, improving efficiency and reducing the need for a full provisioning service on the backend.
+- While MPS and RPS are built using Node.js, which presents dependency management challenges, Console will be built in Go, offering better dependency control.
```

Comment:
This has to do with reducing our codebase that we manage since our team size has been reduced. In fact, i think our console dependency tree is much larger than our MPS or RPS dependency tree. All the tools are automated for dependency management, so regardless, this isn't and won't be a factor in our direction.